src/wireviz/wireviz.py

```diff
@@ -409,14 +409,15 @@ def _get_yaml_data_and_path(inp: Union[str, Path, Dict]) -> (Dict, Path):
             yaml_path = Path(inp).expanduser().resolve(strict=True)
             # if no FileNotFoundError exception happens, get file contents
             yaml_str = open_file_read(yaml_path).read()
-        except (FileNotFoundError, OSError) as e:
-            # if inp is a long YAML string, Pathlib will raise OSError: [errno.ENAMETOOLONG]
-            # (in Windows, it seems OSError [errno.EINVAL] might be raised in some cases)
-            # when trying to expand and resolve it as a path.
-            # Catch this error, but raise any others
+        except (FileNotFoundError, OSError, ValueError) as e:
+            # if inp is a long YAML string, Pathlib will raise OSError [errno.ENAMETOOLONG]
+            # in Windows, ValueError or OSError [errno.EINVAL or None] also might be raised
+            # when trying to expand and resolve it as a path (depending on the Python version)
+            # Catch these specific errors, but raise any others
```

Comment:
I feel your latest rewriting of these comments might have unintentionally changed the meaning a bit, so I suggest a new rewriting that hopefully is easier to understand:
```suggestion
            # if inp is a long YAML string, Pathlib will normally raise
            # FileNotFoundError or OSError(errno = ENAMETOOLONG) when
            # trying to expand and resolve it as a path, but in Windows
            # might ValueError or OSError(errno = EINVAL or None) be raised
            # instead in some cases (depending on the Python version).
            # Catch these specific errors, but raise any others.
```